ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/index.scss

```
@@ -0,0 +1,22 @@
+.advanced-gas-fee-input {
+  a.advanced-gas-fee-input__edit-link {
+    display: inline;
+    font-size: $font-size-h7;
+    padding: 0;
+    white-space: nowrap;
+  }
+
+  .form-field__heading-title > h6 {
+    font-size: $font-size-h7;
+  }
+
+  &__border {
+    padding-bottom: 16px; // or 24px
```

Oh sorry I shouldn't have put that comment in my suggestion. I just meant pick either or because 20px is in the middle.
```suggestion
    padding-bottom: 16px;
```